chsdi/static/doc/examples/ol3_lv95.js

```diff
@@ -0,0 +1,112 @@
+var RESOLUTIONS = [
+    4000, 3750, 3500, 3250, 3000, 2750, 2500, 2250, 2000, 1750, 1500, 1250,
+    1000, 750, 650, 500, 250, 100, 50, 20, 10, 5, 2.5, 2, 1.5, 1, 0.5
+];
+
+var extent = [2420000, 130000, 2900000, 1350000];
+
+function qualifyURL(url) {
+    var a = document.createElement('a');
+    a.href = url;
+    return a.cloneNode(false).href;
+}
+
+var matrixIds = [];
+for (var i = 0; i < RESOLUTIONS.length; i++) {
+    matrixIds.push(i);
+}
+
+
+var wmtsSource = function(layer, options) {
+    var resolutions = options.resolutions ? options.resolutions : RESOLUTIONS;
+    var tileGrid = new ol.tilegrid.WMTS({
+        origin: [extent[0], extent[3]],
+        resolutions: resolutions,
+        matrixIds: matrixIds
+    });
+    var extension = options.format || 'png';
+    var timestamp = options['timestamps'][0];
+    return new ol.source.WMTS(({
+        attributions: [new ol.Attribution({
+            html: '<a target="new" href="http://www.swisstopo.admin.ch/' +
+                'internet/swisstopo/en/home.html">swisstopo</a>'
+        })],
+        url: (qualifyURL('..') + '1.0.0/{Layer}/default/' +
+            timestamp + '/2056/' +
+            '{TileMatrix}/{TileCol}/{TileRow}.').replace('http:', location.protocol) + extension,
+        tileGrid: tileGrid,
+        layer: options['serverLayerName'] ? options['serverLayerName'] : layer,
+        requestEncoding: 'REST'
+    }));
+};
+
+// See http://api3.geo.admin.ch/rest/services/api/MapServer/layersConfig
```

Comment:
I am wondering if we shouldn't simply document this service as it could be useful to many people. (we already gave the tip to a few people via google group.) @cedricmoullet what do you think?